torch/_inductor/ir.py

```diff
@@ -3810,6 +3810,19 @@ def apply_constraint(self):
 
 
 class UserDefinedTritonKernel(ExternKernel):
+    def _remove_constexpr_args(self, args):
```

Comment:
take kernel here from below and make this a static function

you can in fact inline this to codegen since you dont call this function anywhere else

Reply:
Actually after taking the extracted kernel in def codegen, this function is only one line now, please just inline it below instead, no need for new function

Reply:
Few lines, but inlined.